package.json

```diff
@@ -3,19 +3,29 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
+    "cross-env": "^6.0.3",
```

Comment:
Prettier, node-sass, and lint-staged should be devDependencies. They are not needed in prod.